scylla-cql/src/serialize/writers.rs

```diff
@@ -139,20 +156,29 @@ pub struct CellValueBuilder<'buf> {
 
     // Starting position of the value in the buffer.
     starting_pos: usize,
+
+    // Should we write the size of the value?
+    write_size: bool,
 }
 
 impl<'buf> CellValueBuilder<'buf> {
     #[inline]
-    fn new(buf: &'buf mut Vec<u8>) -> Self {
+    fn new(buf: &'buf mut Vec<u8>, write_size: bool) -> Self {
         // "Length" of a [bytes] frame can either be a non-negative i32,
         // -1 (null) or -1 (not set). Push an invalid value here. It will be
```

Comment:
:camping: Let's fix the old typo. -1 is either null or not set, not both of them.

Reply:
-2 corresponds to not set, according to the CQL specs.